modules/dagger/arrow-dagger/src/main/kotlin/arrow/dagger/instances/optiont.kt

```diff
@@ -39,10 +39,12 @@ class DaggerOptionTFunctorInstance<F> @Inject constructor(val FF: Functor<F>) :
 
 class DaggerOptionTApplicativeInstance<F> @Inject constructor(val FF: Monad<F>) : OptionTApplicativeInstance<F> {
   override fun FF(): Monad<F> = FF
+  override fun MF(): Monad<F> = FF
```

Comment:
Duplicated?

Reply:
Different interdependent instances in the hierarchy declared a dependency on the nested monad multiple times with different names. It wasn't in the scope of this massive refactoring to adjust all names in type class hierarchies because they are all wrong and it would have been another massive change